City-of-Pori/pori-kada
# PORI-137: Single news page styling

Single news page styling.

To test:
1. drush updb -y && drush fra -y && drush cc all
2. make sure you have some news items.
3. go to a single news page and confirm that the page looks somewhat similar to a normal page
4. confirm that there is a link to news archive at the top of the page.
